scripts/docscheck.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+docs=$(ls website/docs/**/*.markdown)
+error=false
+
+for doc in $docs; do
+  dirname=$(dirname "$doc")
+  category=$(basename "$dirname")
+
+
+  case "$category" in
+    "guides")
+      # Guides have no requirements
+      continue
+      ;;
+
+    "d")
+      # Data sources have no requirements
```

Comment:
@pwelch datasources do require the subcategory

Reply:
@jotruon Updated the script